packages/celer-web-app/package.json

```diff
@@ -145,6 +145,10 @@
       "jsx",
       "node"
     ],
+    "moduleDirectories": [
+      "node_modules",
+      "src"
+    ],
```

Comment:
What's this for?

Reply:
This allows the testing modules to be visible to Jest. Without this added, I was unable to have have any ``*.Test.*`` files be recognized and run.

Reply:
Works fine for me without it. Did you run it with `npm run test`?

Reply:
My B, I just realized the problem. I have fixed it for you